### PR TITLE
[RFC] ExpressionLanguage check

### PR DESCRIPTION
**Pull request by kbond**
Currently, you can only get ini values via the `ini` function in your expressions.  

Suggestions for other available functions?

### CODE REVIEW

**Comment by lsmith77** [comment]
@kbond can you rebase this and we will then merge it .. once its in the wild, people might come with new ideas.

**Comment by kbond** [comment]
Ok, I rebased but want to add some tests before merging. I will let you know when ready.

**Comment by kbond** [comment]
Should be good to go (assuming travis passes) if you want to give it a final review.

File: composer.json [hunk 1]
```diff
@@ -30,7 +30,8 @@
     "require-dev": {
         "matthiasnoback/symfony-dependency-injection-test": "~0.4",
         "sensiolabs/security-checker": "~1.3",
-        "guzzle/guzzle": "~3.8"
+        "guzzle/guzzle": "~3.8",
+        "symfony/expression-language": "~2.3"
```
Comment (lsmith77) [inline composer.json]: should also be added to the suggest

**Comment by lsmith77** [comment]
awesome!

**Pull request merged**